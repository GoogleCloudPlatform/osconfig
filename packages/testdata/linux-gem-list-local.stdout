
*** LOCAL GEMS ***

benchmark (default: 0.1.0)
bigdecimal (default: 2.0.0)
bundler (default: 2.1.2)
cgi (default: 0.1.0)
csv (default: 3.1.2)
date (default: 3.0.0)
dbm (default: 1.1.0)
delegate (default: 0.1.0)
did_you_mean (default: 1.4.0)
etc (default: 1.1.0)
fcntl (default: 1.0.0)
fiddle (default: 1.0.0)
fileutils (default: 1.4.1)
forwardable (default: 1.3.1)
gdbm (default: 2.1.0)
getoptlong (default: 0.1.0)
io-console (default: 0.5.3)
ipaddr (default: 1.2.2)
irb (default: 1.2.1)
json (default: 2.3.0)
logger (default: 1.4.2)
matrix (default: 0.2.0)
minitest (5.13.0)
mutex_m (default: 0.1.0)
net-pop (default: 0.1.0)
net-smtp (default: 0.1.0)
net-telnet (0.1.1)
observer (default: 0.1.0)
open3 (default: 0.1.0)
openssl (default: 2.1.2)
ostruct (default: 0.2.0)
power_assert (1.1.7)
prime (default: 0.1.1)
pstore (default: 0.1.0)
psych (default: 3.1.0)
racc (default: 1.4.16)
rake (13.0.1)
rdoc (default: 6.2.1)
readline (default: 0.0.2)
readline-ext (default: 0.1.0)
reline (default: 0.1.2)
rexml (default: 3.2.3)
rss (default: 0.2.8)
sdbm (default: 1.0.0)
singleton (0.3.0, default: 0.1.0)
stringio (default: 0.1.0)
strscan (default: 1.0.3)
test-unit (3.3.5)
timeout (default: 0.1.0)
tracer (default: 0.1.0)
uri (default: 0.10.0)
webrick (default: 1.6.0)
xmlrpc (0.3.0)
yaml (default: 0.1.0)
zlib (default: 1.1.0)
