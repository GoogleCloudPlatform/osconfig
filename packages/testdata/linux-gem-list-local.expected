[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "benchmark",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bigdecimal",
        Arch:    "all",
        RawArch: "",
        Version: "2.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bundler",
        Arch:    "all",
        RawArch: "",
        Version: "2.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cgi",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "csv",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "date",
        Arch:    "all",
        RawArch: "",
        Version: "3.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "dbm",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "delegate",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "did_you_mean",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "etc",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fcntl",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fiddle",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "fileutils",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "forwardable",
        Arch:    "all",
        RawArch: "",
        Version: "1.3.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "gdbm",
        Arch:    "all",
        RawArch: "",
        Version: "2.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "getoptlong",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "io-console",
        Arch:    "all",
        RawArch: "",
        Version: "0.5.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ipaddr",
        Arch:    "all",
        RawArch: "",
        Version: "1.2.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "irb",
        Arch:    "all",
        RawArch: "",
        Version: "1.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "json",
        Arch:    "all",
        RawArch: "",
        Version: "2.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "logger",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "matrix",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "minitest",
        Arch:    "all",
        RawArch: "",
        Version: "5.13.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "mutex_m",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-pop",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-smtp",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "net-telnet",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "observer",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "open3",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "openssl",
        Arch:    "all",
        RawArch: "",
        Version: "2.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "ostruct",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "power_assert",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.7",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "prime",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "pstore",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "psych",
        Arch:    "all",
        RawArch: "",
        Version: "3.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "racc",
        Arch:    "all",
        RawArch: "",
        Version: "1.4.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rake",
        Arch:    "all",
        RawArch: "",
        Version: "13.0.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rdoc",
        Arch:    "all",
        RawArch: "",
        Version: "6.2.1",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "readline",
        Arch:    "all",
        RawArch: "",
        Version: "0.0.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "readline-ext",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "reline",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.2",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rexml",
        Arch:    "all",
        RawArch: "",
        Version: "3.2.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "rss",
        Arch:    "all",
        RawArch: "",
        Version: "0.2.8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "sdbm",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "singleton",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "singleton",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "stringio",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "strscan",
        Arch:    "all",
        RawArch: "",
        Version: "1.0.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "test-unit",
        Arch:    "all",
        RawArch: "",
        Version: "3.3.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "timeout",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "tracer",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "uri",
        Arch:    "all",
        RawArch: "",
        Version: "0.10.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "webrick",
        Arch:    "all",
        RawArch: "",
        Version: "1.6.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "xmlrpc",
        Arch:    "all",
        RawArch: "",
        Version: "0.3.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "yaml",
        Arch:    "all",
        RawArch: "",
        Version: "0.1.0",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "zlib",
        Arch:    "all",
        RawArch: "",
        Version: "1.1.0",
        Source:  packages.Source{},
    },
}